Microsoft/WebTemplateStudio
# Add App Service Form

Issue #888 
Task https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30983

**Changes**
- App Service Form should open when 'Add Resource` is clicked on the App Service card
- Form data should be able to be saved and retrieved through the `appService` store

**How to Test**
You should be able to test this through the mock vs code API using the **browser** (the extension side commands `EXTENSION_COMMANDS.NAME_APP_SERVICE` and `EXTENSION_COMMANDS.SUBSCRIPTION_DATA_APP_SERVICE` are not in dev yet)
1) Check that you can open App Service Form through the card
2) Check that you can save and retrieve data you put in to the form
3) Check that runtime stack changes based on backend framework chosen

![image](https://user-images.githubusercontent.com/21695878/61325910-94037e00-a7ca-11e9-8635-945d226becc2.png)
